### PR TITLE
Accounts Sidebar Auto Refresh

### LINKED ISSUE #22: CCDScan: auto refresh in CCDScan Account's sidebar info. doesn't happen, user has to reload CCDScan again in order to view changes
CCDScan: auto refresh in CCDScan Account's sidebar info. doesn't happen, user has to reload CCDScan again in order to view changes

https://stagenet.ccdscan.io/

Change in Account sidebar related info. e.g. new transactions etc. doesn't reflect unless CCDScan page is reloaded again and then account page is reopened. This doesn't look to be a good user experience. New info. should refresh automatically without having to reload CCDScan site.

### PR DESCRIPTION
**Pull request by parv0888**
## Purpose
fix #22 

## Changes
Added Accounts Updated GraphQl Subscription

## Checklist

- [x] My code follows the style of this project.
- [x] The code compiles without warnings.
- [x] I have performed a self-review of the changes.
- [x] I have documented my code, in particular the intent of the
      hard-to-understand areas.
- [x] (If necessary) I have updated the CHANGELOG.

## CLA acceptance

By submitting the contribution I accept the terms and conditions of the
Contributor License Agreement v1.0
- link: https://developers.concordium.com/CLAs/Contributor-License-Agreement-v1.0.pdf

- [x] I accept the above linked CLA.

### CODE REVIEW

File: backend/committed_graphql_schema.graphql [hunk 1]
```diff
@@ -230,6 +230,10 @@ type AccountsMetricsBuckets {
   y_AccountsCreated: [Int!]!
 }
 
+type AccountsUpdatedSubscriptionItem {
+  address: String!
+}
+
```
Comment (abizjak) [inline backend/committed_graphql_schema.graphql:236]: Could we add documentation please

File: backend/Application/Api/GraphQL/Import/ImportWriteController.cs [hunk 2]
```diff
@@ -185,9 +185,25 @@ private async Task<BlockWriteResult> HandleCommonWrites(BlockDataPayload payload
...
 
-        _eventLogHandler.HandleLogs(transactions);
+        var updatedAccountAddresses = accountTokenUpdates
+            .Select(u => u.Address)
+            .Concat(accountBalanceUpdates.Select(a => a.AccountAddress))
+            .Distinct()
+            .ToArray();
+
+        if (updatedAccountAddresses.Length > 0)
+        {
```
Comment (abizjak) [inline backend/Application/Api/GraphQL/Import/ImportWriteController.cs:198]: why is this if necessary? Won't it just be an empty loop otherwise?

**Review by abizjak — COMMENTED** [review]
Have you tested this on a non-trivial amount of updates? Such as 1000 subscribers?

File: backend/Application/Api/GraphQL/Import/ImportWriteController.cs (hunk 2 above)
Comment (parv0888) [reply backend/Application/Api/GraphQL/Import/ImportWriteController.cs]: removed, thanks

**Comment by abizjak** [comment]
Can you please resolve conflicts on this so we can get it merged/?

**Comment by parv0888** [comment]
> Can you please resolve conflicts on this so we can get it merged/?

done

**Review by abizjak — APPROVED** [review]

**Pull request merged**